### ISSUE TITLE
Add functionality listing available rates in the OpenADAS repository

### ISSUE THREAD

**Issue by Mateasek** [cherab/core#230]
I was asked about this functionality a few times and I think it is not a bad idea to add functions which would list available rates without the data. The functions could be placed in cherab.openadas.repository.utility. I already have some code listing ionisation, recombination and radiated power rates [here](https://github.com/Mateasek/cherab/blob/feature/list_available_rates/cherab/openadas/repository/utility.py). In simmilar way it could be broadened to include rest of the rates, which would return a little bit more complicated dictionaries.

**Comment by mattngc** [comment]
Agree, this is a good idea. We need to help users work out what data is available.
